README.md

```diff
@@ -72,9 +72,11 @@ use {
 <td>
 
 ```lua
--- No idea how to set dependency in vim-plug, pls let me know if you use it
--- You need to have "MunifTanjim/nui.nvim" as Dependency
-Plug "better-ts-errors.nvim", { "tag": "*" }
+-- vim-plug no longer handles dependencies between plugins and it's 
+-- up to the user to manually specify Plug commands for dependent plugins.
+-- https://github.com/junegunn/vim-plug/wiki/faq#managing-dependencies 
+Plug "MunifTanjim/nui.nvim"
+Plug "OlegGulevskyy/better-ts-errors.nvim"
```

Comment:
Dependency expression seems to be a matter of taste with vim-plug. [[src](https://github.com/junegunn/vim-plug/wiki/faq#managing-dependencies)]

Some folks use pipe separation:
```vimscript
-- This does not work for lua configs!
Plug 'SirVer/ultisnips' | Plug 'honza/vim-snippets'
```

While some use indentation:
```vimscript
Plug 'kana/vim-textobj-user'
  Plug 'nelstrom/vim-textobj-rubyblock'
  Plug 'whatyouhide/vim-textobj-xmlattr'
  Plug 'reedes/vim-textobj-sentence'
```

I chose to leave this documentation unopinionated but am happy to implement a pattern if there's a desire.